hw01.md

```diff
@@ -0,0 +1,16 @@
+# Lindsay Cooper 
+Hmwk 1
+
+I am a MAPSS student concentrating in psychology. I am especially interested in clinical and community psychology. I recently moved to Chicago from Boston. I am originally from Seattle. I was a preschool teacher for several years prior to joining MAPSS.
+
+My favorite spot in Chicago (so far):
+<em> https://www.artic.edu <em>
```

Comment:
I wasn't sure if you wanted to intentionally show the link but if you wanted to embed the link in a text, you can use the syntax [text](link url). What you used was in the cheat sheet that Ben handed out so really not a problem!